src/shared/components/SharePost.tsx

```diff
@@ -38,11 +49,10 @@ const SharePost = ({ profilePic, sx, onClick, isPost }: SharePostProps) => {
         display: "flex",
         flexDirection: "column",
         justifyContent: "space-between",
-        backgroundColor: "#ffffff",
-        maxWidth: "600px",
-        margin: "auto",
+        backgroundColor: theme.backgroundColor,
```

Comment:
nit: I would destructure these properties from the theme

Reply:
sounds good